lib/src/component/dom_components.dart

```diff
@@ -57,8 +57,6 @@ class SvgProps extends component_base.UiProps with DomPropsMixin, ReactPropsMixi
 
 /// A class that provides namespacing for static DOM component factory methods, much like `React.DOM` in React JS.
 abstract class Dom {
-  /// Returns a new [DomPropsMixin] that renders an `<a>` tag with getters/setters for all DOM-related React props
-  static DomProps a() => new DomProps(react.a);
```

Comment:
@jacehensley-wf why would be changed from `DomProps` to `SvgProps`? _(Same with all other related changes)_

Isn't this a breaking change?

Is it necessary to change the type just so that it can be used within an `<svg>` element?

If its not possible to use `Dom.a` within an SVG context, could we just namespace like this instead?

```dart
abstract class Dom {
  static DomProps a() => new DomProps(react.a);
  static SvgProps svgA() => new SvgProps(react.a);

  static DomProps audio() => new DomProps(react.audio);
  static SvgProps svgAudio() => new SvgProps(react.audio);

  // ... etc ...
}
```

cc/ @greglittlefield-wf @clairesarsam-wf